.appveyor.yml

```diff
@@ -3,7 +3,7 @@ environment:
       PYTHON: "C:\\Python36"
 
     matrix:
-        - TOXENV: py27-pip8.1.1
+        - TOXENV: py27-pip8.1.1-coverage
           PIP: 8.1.1
```

Comment:
Do we really need PIP variable?

Reply:
It is used for matrix expansion, in the codecov name, and in https://github.com/blueyed/pip-tools/blob/3add05df690b4fd3350e1deb48909055897f45b1/tests/test_cli_compile.py#L19-L24.